fsspec/spec.py

```diff
@@ -1170,12 +1170,12 @@ def expand_path(self, path, recursive=False, maxdepth=None, **kwargs):
             raise FileNotFoundError(path)
         return sorted(out)
 
-    def mv(self, path1, path2, recursive=False, maxdepth=None, **kwargs):
+    def mv(self, path1, path2, recursive=False, maxdepth=None, on_error=None, **kwargs):
         """Move file(s) from one location to another"""
         if path1 == path2:
             logger.debug("%s mv: The paths are the same, so no files were moved.", self)
         else:
-            self.copy(path1, path2, recursive=recursive, maxdepth=maxdepth)
+            self.copy(path1, path2, recursive=recursive, maxdepth=maxdepth, on_error=on_error)
             self.rm(path1, recursive=recursive)
```

Comment:
Also pass to `rm` ?

Reply:
I did not see this parameter in the rm methods' definitions. So not sure how to pass that into these.
Likely those never swallow errors, only copy does w recursive, right?